flex-linux-setup/flex_linux_setup/templates/env-config.js

```diff
@@ -0,0 +1,7 @@
+const CONFIG_API_BASE_URL = "https://%(hostname)s/jans-config-api"
+const API_BASE_URL = "https://%(hostname)s/jans-config-api/admin-ui"
+const BASE_PATH = "/admin"
```

Comment:
we can use `BASE_PATH=/admin/` now as the main issue in cloud-native has been resolved.